### DISCUSSION TITLE
Proxy Support

### DISCUSSION THREAD

**Discussion by sebastian-moeckel** [valinet/ExplorerPatcher#2630] [Ideas]
I use the Explorer Patcher at my companies notebook. It would be nice, to have the option, to configure a simple https proxy. 
(To be informed about new updates).

best regards and thank You for that great software!!

**Comment by pyrates999** [comment]
That's tricky because it attempts to download symbols online while you are installing it.  So it's better if it just supported the proxy settings that the settings application uses.

**Comment by pyrates999** [comment]
However, I would not install it on any work computer because it requires elevated privileges to work.  And that can be taken away from you on a work computer at anytime.
